server/controllers/resourceController.js

```diff
@@ -45,24 +44,63 @@ resourceController.createResource = async (req, res, next) => {
       createErr({
         method: 'createResource',
         type: 'data validation error',
-        err: 'request body did contained invalid data',
+        err: 'request body contained invalid data',
       })
     );
   }
-
   try {
     const dbRes = await Resource.create({ id, user, date, text, embeds, attachments, score, resources, subject, category });
     res.locals.newResource = dbRes;
   } catch (err) {
     return next(
       createErr({
-        method: 'createExample',
+        method: 'db create',
         type: 'db insert error',
         err,
       })
     );
   }
+  return next();
+};
 
+resourceController.deleteResource = async (req, res, next) => {
+  const required = ['id', 'user', 'date'];
```

Comment:
For delete, we should only need the unique id of the resource to delete, right?

Reply:
Resolved by #47